app-misc/g810-led/g810-led-0.4.2.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 2018-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils systemd toolchain-funcs udev
+
+DESCRIPTION="Led controller for Logitech G- Keyboards"
+HOMEPAGE="https://github.com/MatMoul/g810-led"
+SRC_URI="https://github.com/MatMoul/g810-led/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+hidapi"
+
+RDEPEND="
+	hidapi? ( dev-libs/hidapi:= )
+	!hidapi? ( virtual/libusb:= )
+	"
+DEPEND="${RDEPEND}"
+
+DOCS=("README.md" "sample_profiles" "sample_effects")
+
+src_prepare() {
+	default
+	# See
+	# https://github.com/systemd/systemd/issues/4288
+	# https://sourceforge.net/p/sigrok/mailman/sigrok-devel/thread/12691365.gQiffmFRoU%40pebbles.site/
+	# We remove the MODE-bit since it's already set to 660+GROUP="input" by default udev rules
+	sed -i \
+		-e 's|MODE="666"|TAG+="uaccess",|' \
+		udev/g810-led.rules || die
+}
+
+src_compile() {
+	emake LIB="$(usex hidapi hidapi libusb)" CXX="$(tc-getCXX)" bin-linked
+}
+
+src_install() {
+	dolib.so "lib/libg810-led.so.${PV}"
+	dosym "libg810-led.so.${PV}" "/usr/$(get_libdir)/libg810-led.so"
+
+	insinto /etc/g810-led/
+	newins sample_profiles/group_keys profile
+	newins sample_profiles/all_off reboot
+
+	dobin bin/g810-led
+	local boards=(213 410 413 512 513 610 815 910 pro)
+	local x
+	for x in "${boards[@]}"; do
+		dosym g810-led "/usr/bin/g${x}-led"
+	done
+
+	insinto /usr/include/g810-led
+	doins src/classes/*.h
```

Comment:
Just pure curiosity to see if I should fix 0.3.6: Is there other programs out there using g810-led as a library?

Reply:
Not to my knowledge